templates/scenarios/js/notification-restify/teamsfx/app.local.yml

```diff
@@ -0,0 +1,29 @@
+version: 1.0.0
+
+deploy:
```

Comment:
Make lifecycles in execution order: registerApp -> configureApp -> deploy.
And, where's `m365bot` action?

Reply:
Reorder the lifecycle.
The `m365bot` will be in another PR.